principles/fp-003-uris.md

```diff
@@ -58,8 +53,5 @@ Counter-Examples
 
 -   There are systems that use alphanumeric id's. This should be
     discouraged, especially as these have semantic content.
--   An example is this NCI class: 'Chemotherapy Regimen or Agent
-    Combination':
-    <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#Chemotherapy_Regimen>
```

Comment:
Why would you remove this, this is not really a URL - its a IRI, so it does not have to resolve to anything, right?

Reply:
We thought it would be better if it referenced a resolvable resource.

Reply:
Ok cool, if you discussed this with the rest, I am in. Thanks for clarification!